src/components/TaxPayer/PersonFields.tsx

```diff
@@ -65,6 +71,11 @@ export const PersonListItem = ({ person, remove }: PersonListItemProps): ReactEl
       primary={`${person.firstName} ${person.lastName}`}
       secondary={person.ssid}
     />
+    <ListItemIcon>
+      <IconButton onClick={edit} edge="end" aria-label="edit">
```

Comment:
If it's undefined let's not display the edit icon for now, so the edit button is not visible for the dependents?